tiatoolbox/utils/visualization.py

```diff
@@ -138,3 +161,45 @@ def overlay_patch_prediction(
     cbar.ax.tick_params(labelsize=12)
 
     return ax
+
+
+def overlay_instance_prediction(
```

Comment:
As a minor suggestion, isn't it better to call this function: `overlay_prediction_contours` and the other one `overlay_prediction_masks` (instead of `overlay_patch_prediction`), or something general like these? because these two function can be used for other tasks as well. For example, we used `overlay_patch_prediction` function for depicting wsi semantic segmentation performance. On the other hand, `overlay_instance_prediction` can also be used to overlay of the contours of semantic segmentation results where `instance` term does not apply.

Reply:
Updated, you will have to change the notebook.